api/src/pcapi/domain/book_types.py

```diff
@@ -73,19 +73,35 @@ class BookType:
             GTL(code="02000000", label="Jeunesse", level=1),
             GTL(code="01060000", label="Biographie / Témoignage littéraire", level=2),
             GTL(code="01040000", label="Contes / Légendes", level=2),
+            GTL(code="92000000", label="Romance", level=1),
+            GTL(code="91000000", label="Fantasy & Science-fiction", level=1),
+            GTL(code="90000000", label="Policier & Thriller", level=1),
         ],
         children=[
-            BookSubType(position=1, label="Romances", gtls=[GTL(code="01020600", label="Roman sentimental", level=3)]),
-            BookSubType(position=2, label="Thriller", gtls=[GTL(code="01020500", label="Thriller", level=3)]),
-            BookSubType(position=3, label="Fantasy", gtls=[GTL(code="01020900", label="Fantasy", level=3)]),
-            BookSubType(position=4, label="Policier", gtls=[GTL(code="01020400", label="Policier", level=3)]),
+            BookSubType(
+                position=1,
+                label="Romances",
+                gtls=[
+                    GTL(code="92010000", label="Romance autre", level=2),
+                    GTL(code="92020000", label="Romance contemporaine", level=2),
+                    GTL(code="92030000", label="Romance suspense", level=2),
+                    GTL(code="92040000", label="Romance historique", level=2),
+                    GTL(code="92050000", label="Romance paranormale", level=2),
+                    GTL(code="92060000", label="Comédie romantique", level=2),
+                    GTL(code="92070000", label="Romance Erotique", level=2),
+                    GTL(code="92080000", label="Dark Romance", level=2),
```

Comment:
```suggestion
                    GTL(code="92000000", label="Romance", level=1),
```
J'ai l'impression qu'il n'y a aucun sous-groupe de "Romance" qu'on veut exclure. Donc on peut se contenter de mettre le GTL parent.